src/expression.d

```diff
@@ -11813,10 +11813,21 @@ public:
  */
 extern (C++) final class CommaExp : BinExp
 {
+    /// This is needed because AssignExp rewrites CommaExp, hence it needs
+    /// to trigger the deprecation.
+    bool isGenerated;
```

Comment:
It looks like this field is never changed after construction. Can you make it `const`?